modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2465,6 +2460,14 @@ private void _deleteFromTable(
 		FinderCacheUtil.clearDSLQueryCache(dbTableName);
 	}
 
+	private void _deleteTempFileEntry() throws PortalException {
```

Comment:
@vngoliveira I think we also need to call this at `addOrUpdateExtensionDynamicObjectDefinitionTableValues`